packages/admin-panel/src/importExport/ExportModal.jsx

```diff
@@ -133,8 +133,8 @@ export const ExportModal = React.memo(
           id="page-export-button"
           startIcon={<ExportIcon />}
           onClick={handleOpen}
-          isLoading={STATUS === STATUS.LOADING}
-          disabled={STATUS === STATUS.ERROR}
+          isLoading={status === STATUS.LOADING}
+          disabled={status === STATUS.ERROR}
```

Comment:
Unrelated to this PR, just spotted the bug while looking for the relevant bits of code